src/ui/layout/Layout.vue

```diff
@@ -45,14 +45,16 @@
     <div class="l-shell__drawer c-drawer c-drawer--push c-drawer--align-top"></div>
 
     <multipane
-        class="l-shell__main"
+        :class="['l-shell__main', isResizing ? 'l-shell__resizing' : '']"
```

Comment:
This is fine and you'll see this a lot in our codebase. We should try to bind dynamic styling to computed class and style objects though so these don't get out of hand. Static classes can be left in their own non bound attribute